Packs/DeveloperTools/Scripts/VerifyIntegrationHealth/VerifyIntegrationHealth.py

```diff
@@ -0,0 +1,54 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+from typing import Dict, Any, Tuple
+import traceback
+
+
+def health_check(health_dict, integration_name: str) -> Tuple[bool, bool]:
+    for _, integration in health_dict.items():
+        if integration.get('brand') == integration_name:
+            return (False, True) if integration.get('lastError') else (True, True)
+    return True, False
+
+
+def health_check_command(args: Dict[str, Any]) -> CommandResults:
+
+    integration_name = args.get('integration_name', '')
+
+    raw_result = demisto.executeCommand(
+        "demisto-api-post",
+        {
+            "uri": "/settings/integration/search",
+            "body": {
+                "size": 10,
+                "query": "name:" + integration_name
+            },
+        })
+    health_dict = raw_result[0]["Contents"]["response"]["health"]
```

Comment:
please wrap this with error handling: use `is_error()` and `get_error()` CSP functions (also in other uses of executeCommand in the PR).